test/cluster/mzcompose.py

```diff
@@ -205,7 +205,7 @@ def workflow_test_github_12251(c: Composition) -> None:
     c.down(destroy_volumes=True)
     c.up("materialized")
 
-    start_time = time.process_time()
+    start_time = time.time()
```

Comment:
process_time doesn't include the time waiting for Mz to return results: https://docs.python.org/3/library/time.html#time.process_time